cmd/githubgen/allowlist.txt

```diff
@@ -0,0 +1,26 @@
+Caleb-Hurshman
+Doron-Bargo
+MaxKsyunz
+MitchellGale
+YANG-DB
+ag-ramachandran
+agoallikmaa
+alexvanboxel
+architjugran
+asaharn
+avadhut123pisal
+billmeyer
+eedorenko
+emreyalvac
+keep94
+kiranmayib
+kkujawa-sumo
+leonsp-ai
+liqiangz
+mrsillydog
```

Comment:
This doesn't have to be in this PR but would it be difficult to cross reference the allow list with the member list? I see that mrsillydog is listed in both and is a member. It would be nice if we had a mechanism to ensure we are keeping this list to a minimum.

Reply:
I ended up pushing that change into this PR.